src/modules/auto-scroll-to-id.js

```diff
@@ -27,8 +27,10 @@
 		}
 		
 		if (!!offsetSelector) {
-			var offsetItem = $(offsetSelector).eq(0);
-			offset = offsetItem.height() * -1;
+			var offsetItem = $(offsetSelector).each(function () {
+				var t = $(this);
```

Comment:
@DeuxHuitHuit/review Should we discourage `this` and prefer declaring the function `function (i, e)` and use `$(e)` ? I hate `this`.